src/main/java/pl/edu/mimuw/Main.java

```diff
@@ -1,8 +1,50 @@
 package pl.edu.mimuw;
 
+import pl.edu.mimuw.logic.FormulaFactory;
+import pl.edu.mimuw.logic.Solver;
+import pl.edu.mimuw.logic.Valuation;
+
 public class Main {
 
+  private static void print(String[] var, boolean[] val) {
+    for (int i = 0; i < var.length; i++) {
+      System.out.println(var[i] + "->" + val[i]);
+    }
+  }
+
   public static void main(String[] args) {
-    throw new IllegalStateException("TODO: show formulas pretty-printed examples");
+    var f = new FormulaFactory();
+    var andForm = f.and(f.var("l"), f.var("r"));
```

Comment:
jeżeli chciałes używać tak krótko i ładnie tych funckji, to można je importować statycznie przez `import static` i powinny one zostać raczej jako statyczne w tym factory